config/routes.rb

```diff
@@ -1,8 +1,14 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  mount_devise_token_auth_for "User", at: "auth"
+  mount_devise_token_auth_for 'User', at: 'auth'
   namespace :api, defaults: { format: :json } do
     namespace :v1 do
+      resources :conversations do
+        post 'add_user/:id', to: 'conversations#add_user'
```

Comment:
let's try to change `:id` to `:user_id`
This more likely will let us remove this line
https://github.com/iTechArt-RubyLab/social-network/pull/27/files#diff-0e340663d722a1789e1310ccebdb7c929615b1288cf7063d44c273c4ad0e9735R79